tooling/cli/src/init.rs

```diff
@@ -130,29 +130,41 @@ impl Options {
       )
     })?;
 
-    self.before_dev_command = self
-      .before_dev_command
-      .map(|s| Ok(Some(s)))
-      .unwrap_or_else(|| {
-        prompts::input(
-          "What is your frontend dev command?",
-          Some("npm run dev".to_string()),
-          self.ci,
-          true,
-        )
-      })?;
+    let package_manager =
+      if self.before_build_command.is_none() && self.before_dev_command.is_none() {
+        prompts::select(
+          "What package manager are you using?",
+          &["npm", "pnpm", "yarn", "bun"],
+          Some(0),
+        )?
+        .unwrap()
+      } else {
+        "npm".to_string()
+      };
 
-    self.before_build_command = self
-      .before_build_command
-      .map(|s| Ok(Some(s)))
-      .unwrap_or_else(|| {
-        prompts::input(
-          "What is your frontend build command?",
-          Some("npm run build".to_string()),
-          self.ci,
-          true,
-        )
-      })?;
+    self.before_dev_command =
+      self
+        .before_dev_command
+        .map(Some)
+        .unwrap_or(match package_manager.as_str() {
+          "npm" => Some("npm run dev".to_string()),
```

Comment:
using `dev` and `build` as defaults isn't desired.. some frontends use `start` instead of `dev` and if you have a monorepo you might have a `frontend:dev` script instead...

Reply:
Yes, you're right. This is something I owe a lot of thought to; I'll set a default command that corresponds to the package manager, but still prompt the user for the exact command.